tests/CommunityToolkit.Mvvm.SourceGenerators.Roslyn401.UnitTests/Test_FieldReferenceForObservablePropertyFieldFixer.cs

```diff
@@ -0,0 +1,275 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.ComponentModel;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Test = Microsoft.CodeAnalysis.CSharp.Testing.CSharpCodeFixTest<
+    CommunityToolkit.Mvvm.SourceGenerators.FieldReferenceForObservablePropertyFieldAnalyzer,
+    CommunityToolkit.Mvvm.Fixers.FieldReferenceForObservablePropertyFieldFixer,
+    Microsoft.CodeAnalysis.Testing.Verifiers.MSTestVerifier>;
+using VerifyCS = Microsoft.CodeAnalysis.CSharp.Testing.CSharpCodeFixVerifier<
+    CommunityToolkit.Mvvm.SourceGenerators.FieldReferenceForObservablePropertyFieldAnalyzer,
+    CommunityToolkit.Mvvm.Fixers.FieldReferenceForObservablePropertyFieldFixer,
+    Microsoft.CodeAnalysis.Testing.Verifiers.MSTestVerifier>;
+
+namespace CommunityToolkit.Mvvm.SourceGenerators.Roslyn401.UnitTests;
+[TestClass]
+public class Test_FieldReferenceForObservablePropertyFieldFixer
```

Comment:
I think existing tests for the analyzer should be extended to also test the codefix instead of adding new tests.

Reply:
Only if I felt like dealing with the large msbuild mess in this repo. I didn't 🙂. If sergio wants to move these, he's welcome to.

Reply:
The repo build setup seemed well organized to me 🥲
That hurts...